hooks/useERC721Transfer.ts

```diff
@@ -0,0 +1,26 @@
+import { useCallback } from "react"
+import { transferFromERC721 } from "../lib/cre8or"
+import { useEthersSigner } from "./useEthersSigner"
+
+const useERC721Transfer = ({ afterTransfer }) => {
+  const signer = useEthersSigner()
+
+  const transferERC721 = useCallback(
+    async (erc721Address: string, from: string, to: string, tokenId: number) => {
+      if (!signer) return
+
+      const response = await transferFromERC721(erc721Address, from, to, tokenId, signer)
```

Comment:
can you remove the lib `transferFromERC721` from `../lib/cre8or` and just define the `transferFromERC721` function here in the `useERC721Transfer` hook?

Reply:
moved! 👍